sdk/go/pulumi-language-go/main.go

```diff
@@ -86,9 +92,11 @@ func compileProgramCwd(buildID string) (string, error) {
 func main() {
 	var tracing string
 	var binary string
+	var buildTarget string
 	var root string
 	flag.StringVar(&tracing, "tracing", "", "Emit tracing to a Zipkin-compatible tracing endpoint")
 	flag.StringVar(&binary, "binary", "", "Look on path for a binary executable with this name")
+	flag.StringVar(&buildTarget, "buildTarget", "", "Path to use to output the compiled Pulumi Go program")
```

Comment:
I thought the plan was to make this a boolean flag and to re-use the binary option for the path of where to save this file?

Reply:
During ideation we decided against this approach as it creates a new flag and adds extra complexity around the behavior of a flag when this approach also creates a flag, but doesn't affect existing behavior.